datadog/resource_datadog_synthetics_test_test.go

```diff
@@ -30,33 +30,6 @@ func TestAccDatadogSyntheticsAPITest_importBasic(t *testing.T) {
 				ResourceName:      "datadog_synthetics_test.foo",
 				ImportState:       true,
 				ImportStateVerify: true,
-				// Assertions will be imported into the new schema by default, but we can ignore them as users need to update the local config in this case
-				ImportStateVerifyIgnore: []string{"assertions", "assertion", "options", "options_list"},
-			},
-		},
-	})
-}
-
-func TestAccDatadogSyntheticsAPITest_importBasicNewAssertionsOptions(t *testing.T) {
```

Comment:
Are we covering this import elsewhere?

Reply:
Deprecated field cannot be imported, as the implementation of read prefers the new fields when not set in config (which is the case for import)
I modified the main test config to not have any deprecated fields in it, so the other import tests now does the proper verifications (without the importIgnore)
This one is then rendered useless, as it's just a duplication